allauth/account/tests.py

```diff
@@ -27,7 +27,11 @@
 from .utils import url_str_to_user_pk, user_pk_to_url_str
 
 import uuid
-import mock
+
+try:
+    from unittest import mock
+except ImportError:
+    import mock
```

Comment:
@pennersr this isn't an issue for Travis CI (probably because `mock` library is installed there), but if you try with a clean Python 3 environment, `mock` can't be imported. See also `setup.py`, lines 103-107.